#request

float64 x
float64 y
float64 z
---

# response


float64 x_current
float64 y_current
float64 z_current
